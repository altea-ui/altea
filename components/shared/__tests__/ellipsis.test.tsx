import React from 'react'
import { mount } from 'enzyme'
import Ellipsis from '../ellipsis'

describe('Ellipsis', () => {
  it('should render correctly', () => {
    const wrapper = mount(
      <div style={{ width: '1px' }}>
        <Ellipsis height="10px">text</Ellipsis>
      </div>,
    )
    expect(wrapper.html()).toMatchSnapshot()
    expect(() => wrapper.unmount()).not.toThrow()
  })
})
