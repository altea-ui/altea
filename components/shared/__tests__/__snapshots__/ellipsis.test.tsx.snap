// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ellipsis should render correctly 1`] = `
"<div style=\\"width: 1px;\\"><span>text<style>
        span {
          overflow: hidden;
          text-overflow: ellipsis;
          white-space: nowrap;
          line-height: 10px;
          min-width: 0;
        }
      </style></span></div>"
`;
